analyzer/tools/compilation-database-transformer/tests/functional/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
```

Comment:
Please update the license header in each source file. For more information see: #2697